include/oneapi/dpl/pstl/hetero/dpcpp/sycl_defs.h

```diff
@@ -82,12 +82,12 @@
 #    define _ONEDPL_SYCL_REQD_SUB_GROUP_SIZE_IF_SUPPORTED(SIZE)
 #endif
 
-// The unified future supporting USM host memory and buffers is only supported after DPCPP 2023.1
+// The unified future supporting USM memory and buffers is only supported after DPCPP 2023.1
 // but not by 2023.2.
 #if (_ONEDPL_LIBSYCL_VERSION >= 60100 && _ONEDPL_LIBSYCL_VERSION != 60200)
-#    define _ONEDPL_SYCL_USM_HOST_PRESENT 1
+#    define _ONEDPL_SYCL_UNIFIED_USM_BUFFER_PRESENT 1
```

Comment:
What if a device doesn't  support  USM?

Reply:
The existing code is unchanged where we check if the device supports USM and fall back to a buffer (https://github.com/oneapi-src/oneDPL/blob/main/include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h#L528-L541). The main change here is that older compilers don't support the unified approach in `__usm_host_or_buffer_accessor` at all and we use a regular SYCL accessor instead.